decentraland/unity-renderer
# Chore: cap scene load radius to 4

## What does this PR change?

Fix #2220 
Caps the maximum scene load radius to 4 on the webclient

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/cap-scene-rad
2. Change scene loading radius, verify the max value and that with the max value it's not easy to make the client crash

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
